osv/ecosystems/_ecosystems.py

```diff
@@ -111,6 +112,10 @@ def get(name: str) -> Ecosystem:
     # TODO(michaelkedar)
     return OrderingUnsupportedEcosystem()
 
+  if name.startswith('Ubuntu:'):
+    # TODO(michaelkedar)
```

Comment:
Maybe don't namedrop Michael in the copypasta?

Reply:
Done, I've replaced all occurrences with "unassigned", which is much more accurate